src/boot/efi/devicetree.c

```diff
@@ -9,8 +9,9 @@
 #define FDT_V1_SIZE (7*4)
 
 static void *get_dtb_table(void) {
+        EFI_GUID guid = EFI_DTB_TABLE_GUID;
```

Comment:
Is the memcmp actually more efficient from declaring a variable?  I kind of doubt it.

Also, opportunity to switch this function to size_t.